README.md

```diff
@@ -170,8 +170,22 @@ The second one is set per individual HTTP requests by HTTP client. Its default v
 Flink's current implementation of `AsyncTableFunction` does not allow specifying custom logic for handling Flink AsyncIO timeouts as it is for Java API.
 Because of that, if AsyncIO timer passes, Flink will throw TimeoutException which will cause job restart.
 
-The HTTP request timeouts on the other hand will not cause Job restart. In that case, exception will be logged into application logs.
-To avoid job restart on timeouts caused by Lookup queries, the value of `gid.connector.http.source.lookup.request.timeout` should be smaller than `table.exec.async-lookup.timeout`.
+#### Retries
+Lookup source handles auto-retries for two scenarios:
+1. IOException occurs (e.g. timeout)
+2. HTTP server returns response with code, which is marked as temporal error. These codes are defined in table configuration.
+Retries are executed silently, without job restart. After reaching max retries attempts (per request) function will fail and restart job.
+
+Notice that response codes are splitted into 3 groups:
+- successful responses - response is returned immediately for further processing
+- temporary errors - request will be re-sent
+- error responses - unexpected response, which will fail the job. Any code which is not marked as successful or temporary error is marked as error. 
```

Comment:
unexpected response -> unexpected responses are not retried.

when you say "Any code which is not marked as successful or temporary error is marked as error. " would it be better to say  "Any HTTP error code which is not configured as successful or temporary error is treated as an unretriable error. "